components/eamxx/src/share/io/scream_output_manager.hpp

```diff
@@ -67,46 +67,35 @@ class OutputManager
   using globals_map_t = std::map<std::string,ekat::any>;
 
   // Constructor(s) & Destructor
-  OutputManager () = default;
+  OutputManager() = default;
   virtual ~OutputManager ();
 
-  // Set up the manager, creating all output streams. Inputs:
+  // Initialize
```

Comment:
It may be a good idea to add a comment on what's the main diff between initialize and setup. A one liner is prob enough, something like "initialize sets up members that only depend on runtime parameters, while setup requires grids/fields data and sets up the internal output structures". Badly worded, so I hope you can phrase it better than me.